tutorials/cyber/flow_classification/flow_classification_rapids.ipynb

```diff
@@ -0,0 +1,1322 @@
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "cdf_msk = np.random.rand(len(one_hour_time_bin_cdf)) < 0.7\n",
+    "\n",
+    "train_cdf = one_hour_time_bin_cdf[cdf_msk]\n",
+    "test_cdf = one_hour_time_bin_cdf[~cdf_msk]\n",
+    "\n",
+    "train_X = train_cdf[['pkts','ip_bytes']]\n",
+    "train_Y = train_cdf[['category_id']]\n",
+    "\n",
+    "test_X = test_cdf[['pkts','ip_bytes']]\n",
+    "test_Y = test_cdf[['category_id']]\n",
+    "\n",
+    "xg_train = xgb.DMatrix(train_X, label=train_Y)\n",
+    "xg_test = xgb.DMatrix(test_X, label=test_Y)\n",
+    "\n",
+    "param = {}\n",
+    "param['objective'] = 'multi:softprob'\n",
+    "param['eta'] = 0.1\n",
+    "param['max_depth'] = 8\n",
+    "param['silent'] = 1\n",
+    "param['nthread'] = 4\n",
+    "param['num_class'] = num_categories\n",
+    "param['n_gpus'] = 1\n",
+    "param['tree_method'] = 'gpu_hist'\n",
+    "\n",
+    "watchlist = [(xg_train, 'train'), (xg_test, 'test')]\n",
+    "num_round = 20"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Train the model."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "bst = xgb.train(param, xg_train, num_round, watchlist)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Okay, so we have our new model. We now take some steps to make sure the data is in a format that makes sklearn happy. First we'll use the `predict` function to compute the probabilities. To extend `roc_curve` to multiclass, we'll also need to binarize the labels. Let's keep our sanity by also making sure the lengths match."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "len(bst.predict(xg_test))"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "probs = bst.predict(xg_test).reshape(test_Y.shape[0],param['num_class'])"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "For now, we need to convert the `test_Y` cuDF to an array. The most straight-forward way to do that is to go through Pandas. It also lets us show off how nicely we can convert to Pandas, should the need arise."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "test_Y_binarize = label_binarize(test_Y.to_pandas()['category_id'].values, classes=np.arange(param['num_class']))\n",
+    "\n",
+    "print(\"==> length of probs =\",len(probs))\n",
+    "print(\"==> length of test_Y_binarize =\", len(test_Y_binarize))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Some more housekeeping. We'll create Python dictionaries to hold FPR, TPR, and AUC values."
```

Comment:
I think it would help the user if we added here the expanded forms for TPR, FPR and AUC in brackets

Reply:
Done, and linked to appropriate references.